lldb/source/Symbol/CompilerType.cpp

```diff
@@ -1105,7 +1105,7 @@ bool CompilerType::GetValueAsScalar(const lldb_private::DataExtractor &data,
       return false;
 
     std::optional<uint64_t> byte_size = GetByteSize(exe_scope);
-    if (!byte_size)
+    if (!byte_size || *byte_size == 0)
```

Comment:
Nit: I would add a comment with brief context on how this appears invalid. Otherwise LGTM